package/kernel/lantiq/ltq-vdsl-fw/src/w921v_fw_cutter.c

```diff
@@ -83,8 +88,13 @@ int main(int argc, char **argv)
 	}
 
 	buf_orig = malloc(s.st_size);
+	if (!buf_orig) {
+		printf("Failed to alloc %d bytes\n", s.st_size);
```

Comment:
That should be %zu I believe.

Reply:
The original code used `%d` but I can change that.
Also should I keep the new if I created? Or just keep it as it was originally?

Reply:
I don't see why not.

Reply:
Ok :+1: 
I'm gonna change the string format